src/librustdoc/html/static/main.js

```diff
@@ -2772,38 +2769,6 @@ function hideThemeButtonState() {
         }
     }());
 
-    function createToggleWrapper(tog) {
```

Comment:
Isn't it still in use for all items with attributes?

Reply:
Nope, @jsha cleaned it all up :smile:

Reply:
Nah, @jsha cleaned it up!

Reply:
Makes me even happier. :D